AAI/AAIConnectProfile.md

```diff
@@ -423,7 +423,7 @@ This profile is agnostic to the format of the id_token.
 
 #### Access_token issued by broker
 
-Header - The `kid` parameter must be included and `alg` must be `RS256`.
+Header - The `kid` parameter must be included and `alg` must be `RS256`. Optionally, [jku](https://tools.ietf.org/html/rfc7515#section-4.1.2) can also be included.
```

Comment:
I suggest doing the opposite, and indicating: "[jku](https://tools.ietf.org/html/rfc7515#section-4.1.2) MUST NOT be included".

Then on line 332, add 2.1.1.3 as: "Claim Clearinghouse MUST check that "[jku](https://tools.ietf.org/html/rfc7515#section-4.1.2)" is NOT present, and if present, MUST return an error with indicating that "jku" is not permitted."

Alternatively, if "jku" is kept and Clearinghouse 2.1.1 is updated accordingly to allow for this third model of key exchange, then we would need to add a check requirement somewhere that the "jku" URL is known to the clearinghouse, and the clearinghouse checks the "iss" is expected to have this "jku" as part of the broker trust model.